### DISCUSSION TITLE
Always no result when want play music

### DISCUSSION THREAD

**Discussion by calvgrant** [Androz2091/discord-player#610] [Q&A]
Why Be like this maybe know 
![IMG_20210723_190711](https://user-images.githubusercontent.com/50444758/126773716-e553b5c6-7a9f-4dba-b458-76e6abeceba0.jpg)
Is this a bug or what

**Comment by ustokan** [comment]
Can you show code?

**Reply by calvgrant** [reply]
What code uhh the play or code in main file sorry idk what you mean

**Reply by ustokan** [reply]
Play command code

**Reply by Bardia-SG** [reply]
im facing same problem and play command code is this:

`try {
                                if (!client.player.getQueue(message.guild.id)) {
                                    const song = await client.player.search(wantedMusic, {
                                        requestedBy: message.author
                                    });
                                    if (song.playlist == null && song.tracks.length < 1) return message.channel.send({
                                        content: `${client.emotes.error} - ERROR ON PLAY: No result`
                                    })
                                    const queue = client.player.createQueue(message.guild, {
                                        metadata: wantedMusic,
                                        initialVolume: "100",
                                        autoSelfDeaf: true,
                                        bufferingTimeout: 1000
                                    });
                                    try {
                                        await queue.connect(message.member.voice.channel);
                                    } catch {
                                        message.reply("Could not join your voice channel");
                                    }

                                    wantedMusic.split(' ').slice(0, 1).toString().match(/https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/) ? await queue.addTracks(song.tracks) : await queue.addTrack(song.tracks[0]);


                                    await queue.play();
                                    let botUser = await message.guild.members.cache.get(client.user.id)
                                    if (message.guild.me.voice.channel && botUser.voice.deaf)
                                        botUser.voice.setDeaf(false)
                                    if (message.guild.me.voice.channel && botUser.voice.mute)
                                        botUser.voice.setMute(false)

                                } else {
                                    const queue = client.player.getQueue(message.guild.id)
                                    const song = await client.player.search(wantedMusic, {
                                        requestedBy: message.author
                                    });
                                    wantedMusic.split(' ').slice(0, 1).toString().match(/https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/) ? await queue.addTracks(song.tracks) : await queue.addTrack(song.tracks[0]);
                                }`
